packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -34,6 +34,58 @@ function checkHighlightIs(listbox: HTMLElement, expected: string | null) {
   }
 }
 
+type TestOnScrollToBottom = {
+  reason: 'mouse' | 'keyboard';
```

Comment:
In the last release `touch` was also added. So can you also test for touch devices?

Reply:
>  touch is replacement of click events

How so? You can use `fireEvent.touchStart`.